cime_config/testdefs/testlist_clm.xml

```diff
@@ -120,14 +120,13 @@
       <option name="comment">CESM3 development Exact restart test with change in the processor count at f10, for izumi nag</option>
     </options>
   </test>
-  <!-- this will be removed when CESM no longer needs it-->
-  <test name="ERP_D_Ld9" grid="ne30pg3_t232" compset="I1850Clm51BgcCropG" testmods="clm/clm51cam6LndTuningMode">
+  <!-- Remove when CESM no longer needs it-->
```

Comment:
Just remove this comment about removing when CESM no longer needs it.

Reply:
I think this one is duplicated, so remove the duplicate here.

Reply:
Looking at all the ne30pg3_t232 tests, I did not encounter an exact duplicate, so I'm leaving them unchanged.